### PR TITLE
[EasyStandard] Apply easy Rector rules on the monorepo itself

### PR DESCRIPTION
**Pull request by TomasVotruba**
I've noticed by accident there is bunch of EasyStandard Rector rules that are not applied to itself.

**Let's eat :dog: own :foot:** - this PR fixed it :) 

<br>

I also fixed "inheritdoc" with `.` issue

### CODE REVIEW

File: packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Filter/VirtualSearchFilter.php [hunk 1]
```diff
@@ -10,16 +10,16 @@
...
  *
  * ApiFilter(
  *     VirtualSearchFilter::class,
- *     properties={
- *         "number_partial": {"number": "ipartial"}, --> Virtual search property to number partial
- *         "number_exact": {"number": "exact"}, --> Virtual search property to number exact
- *         "email": "ipartial" --> Normal search property definition
- *     }
- * )
+ *     properties={.
```
Comment (natepage) [inline packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Filter/VirtualSearchFilter.php:17]: @TomasVotruba what about those dots? 😄

**Review by natepage — CHANGES_REQUESTED** [review]

File: packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Filter/VirtualSearchFilter.php (hunk 1 above)
Comment (TomasVotruba) [reply packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Filter/VirtualSearchFilter.php:17]: You should not write broken rules :D  I'll check it.

Comment (natepage) [reply packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Filter/VirtualSearchFilter.php:17]: Is that us!? 😄 We can get someone to check then

Comment (TomasVotruba) [reply packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Filter/VirtualSearchFilter.php:17]: I'll handle it :)

I need someone to merge my other PRs here, so it does not stack/stuck

Comment (natepage) [reply packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Filter/VirtualSearchFilter.php:17]: Yes no worries, I will merge everything today

**Comment by TomasVotruba** [comment]
@natepage Nathan, could you link me the removed method comment you wrote today? I check in on the phone, but I can't find it now on the PC :D

**Comment by natepage** [comment]
@TomasVotruba I've created an issue on rector repo

**Comment by TomasVotruba** [comment]
Ready to merge :heavy_check_mark:

**Comment by TomasVotruba** [comment]
:+1:

**Pull request merged**